### DISCUSSION TITLE
Mute announcements from starred repositories

### DISCUSSION THREAD

**Discussion by OverLordGoldDragon** [community/community#123384] [Other Feature Feedback, Questions, & Ideas]
I'd like to request an option to mute announcements from starred repositories without unstarring them and without selecting "less content like this". I don't want to unstar, and I do want to see announcements from some repos, just not all.

**Comment by karthickkumarsm** [comment]


**Reply by OverLordGoldDragon** [reply]
Seems AI-generated.
